src-tauri/src/bidder/mod.rs

```diff
@@ -58,10 +49,7 @@ impl Bidder {
 
     pub async fn update_bot_if_needed(app: &AppHandle, ssh: &SSH) -> Result<()> {
         if Self::needs_new_bot_version(ssh, app).await? {
-            ssh.run_command("mkdir -p commander-config/bot")
-                .await?;
-            ssh.upload_directory(app, "bot".into()).await?;
-            ssh.upload_directory(app, "bot/src".into()).await?;
+            ssh.upload_directory(app, "bot", "commander-config").await?;
             Self::start_docker(ssh).await?;
```

Comment:
What is going on here? Why remove mkdir? Why not upload bot/src?

Reply:
Ssh upload is recursive now.